packages/modules/cookies/README.md

```diff
@@ -0,0 +1,11 @@
+# Browser Cookie Module
```

Comment:
Please add a short description of what this module should do and how to build it.